### PR TITLE
license: use shelljs for exec sync

### PR DESCRIPTION
**Pull request by sam-github**
/cc @superkhau

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "normalize-package-data": "^2.3.2",
     "request": "^2.49.0",
     "semver": "^5.0.1",
+    "shelljs": "^0.5.3",
```
Comment (rmg) [inline package.json]: no. just no.

**Comment by superkhau** [comment]
LGTM

**Comment by rmg** [comment]
Sorry for the knee-jerk reaction to shelljs.. too many painful memories. After looking at the rest of lib/license.js, it's pretty clear that it would be more than a 10 minute job to rewrite it to be async, so...

Given that I/we went to the trouble of encouraging @gvarsanyi to change [sync-exec](https://www.npmjs.com/package/sync-exec) to an MIT license, I think we should use it instead of shelljs here.

**Comment by sam-github** [comment]
sounds good, I  looked for a smaller sync, but bogged down in the crap npmjs.org search interface, I'll use sync-exec

**Comment by sam-github** [comment]
PTAL

**Comment by superkhau** [comment]
LGTM

**Comment by rmg** [comment]
@sam-github need to modify `index.js` to remove the conditional that is checking for `child_process.execSync`, otherwise `slt license` will still require node v0.12+ (but for no reason).

File: index.js [hunk 2]
```diff
@@ -5,10 +5,7 @@ module.exports = {
   cla: require('./lib/cla'),
   Project: require('./lib/project'),
   info: require('./lib/info'),
+  license:  require('./lib/license');
```
Comment (rmg) [inline index.js]: s/;/,/

**Comment by rmg** [comment]
LGTM, :shipit:

**Pull request merged**